### PR TITLE
removed hard-coded endpoint postings

### PR DESCRIPTION
**Pull request by g0dReVeN**

### CODE REVIEW

File: config/peachpayment.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-if (config('app.env') === 'production') {
+if (in_array(config('app.env'), ['production', 'uat'])) {
```
Comment (rian) [inline config/peachpayment.php]: I don't think we should include this - it's quite specific to our project

File: src/Api/Factory/PaymentScheme.php [hunk 2]
```diff
@@ -63,7 +63,7 @@ protected function getApiUri(bool $version = true): string
          * testMode=INTERNAL causes transactions to be sent to our simulators,
          * which is useful when switching to the live endpoint for connectivity testing.
          * */
-        if (Config::get('app.env') !== 'production') {
+        if (in_array(Config::get('app.env'), ['production', 'uat'])) {
```
Comment (rian) [inline src/Api/Factory/PaymentScheme.php]: This looks incorrect

File: config/peachpayment.php (hunk 1 above)
Comment (g0dReVeN) [reply config/peachpayment.php]: Suspected as much

File: src/Api/Factory/PaymentScheme.php (hunk 2 above)
Comment (g0dReVeN) [reply src/Api/Factory/PaymentScheme.php]: Good catch, thanks

**Review by rian — APPROVED** [review]

**Pull request merged**